requirements.txt

```diff
@@ -1,5 +1,7 @@
 boto3
 pandas
-phonenumbers
 pytest
-numpy
\ No newline at end of file
+numpy
+datetime
+setuptools
```

Comment:
Some of these are base python packages so should not be specified in requirements.